pom.xml

```diff
@@ -26,7 +26,7 @@
 
     <groupId>org.aerogear.unifiedpush</groupId>
     <artifactId>admin-ui</artifactId>
-    <version>1.2.6-SNAPSHOT</version>
+    <version>1.3.0-SNAPSHOT</version>
```

Comment:
nope.  last release is 1.2.5, when releasing the maven-release-plugin was bumping it to next iteration, 1.2.6-SNAPSHOT.

Once we release this, the tool (maven-release-plugin) does bump to 1.2.7-SNAPSHOT